src/dev/impl/DevToys/ViewModels/Tools/Text/XmlValidator/Parsing/XsdParser.cs

```diff
@@ -12,12 +12,15 @@ public XsdParser(string source) : base(source)
         {
         }
 
-        internal override XsdParsingResult Parse(string xsdContent)
+        protected override XsdParsingResult ParsingOperation(string xsdContent)
         {
             XmlSchema xmlSchema;
+            string? targetNamespace;
             try
             {
-                xmlSchema = XmlSchema.Read(new XmlTextReader(new StringReader(xsdContent)), ValidationErrorCallBack);
+                StringReader reader = new(xsdContent);
```

Comment:
Make sure to `Dispose()` the reader.

Reply:
added a `using` statement here - does this work for you?